python/tests/test_approximate_nearest_neighbors.py

```diff
@@ -312,6 +453,6 @@ def test_ivfflat_wide_matrix(
     import time
 
     start = time.time()
-    test_ivfflat(combo=combo, data_shape=data_shape, data_type=data_type)
+    test_ann_algorithm(combo=combo, data_shape=data_shape, data_type=data_type)
```

Comment:
I'm not seeing the algorithm field for the combo params for this test.  How could this pass?

Reply:
This does not test ivfpq. It tests the performance of multi_cols input format, and is part of a previous PR: https://github.com/NVIDIA/spark-rapids-ml/pull/691

Reply:
But it now is changed to go via test_ann_algorithm which seems to expect combo[0] to be the algorithm

Reply:
Good finding! This is a slow test so was not run on pre-merge ci. Revised.